sdk/cosmos/azure-cosmos/src/test/java/com/azure/cosmos/CosmosItemTest.java

```diff
@@ -200,6 +200,70 @@ public void queryItemsWithContinuationTokenAndPageSize() throws Exception{
 
     }
 
+    @Test(groups = { "simple" }, timeOut = TIMEOUT)
+    public void readAllItemsOfLogicalPartition() throws Exception{
+        String pkValue = UUID.randomUUID().toString();
+        InternalObjectNode properties = getDocumentDefinition(UUID.randomUUID().toString(), pkValue);
```

Comment:
unfortunately some of our existing tests rely on InternalObjectNode which is an internal class. 
Customers see this and may use this as sample.

Please consider replacing InternalObjectNode with ObjectNode for the newly added tests (as this might get used as sample by end users.)

Reply:
Fixed